packages/framer-motion/src/gestures/use-hover-gesture.ts

```diff
@@ -28,19 +28,27 @@ export function useHoverGesture({
     whileHover,
     visualElement,
 }: FeatureProps) {
+    /**
+     * TODO: Make passive: false if onHoverStart doesnt exist
```

Comment:
What is this TODO for? Shouldn't it be `passive: true` if onHoverStart doesn't exist?

Reply:
Ah we are - I forgot to get rid of the TODOs!